packages/react-icons/convert.js

```diff
@@ -10,6 +10,7 @@ const _ = require("lodash");
 
 const SRC_PATH = argv.source;
 const DEST_PATH = argv.dest;
+const RTL_FILTER_PATH =argv.filter;
```

Comment:
`filter` is a confusing name for this arg. How about `flip_in_rtl`?

Ditto for the file name: `flip_in_rtl.txt` instead of `rtl.txt`

Reply:
Possibly even autoflip_in_rtl